docs/sources/reference/run.md

```diff
@@ -348,6 +349,14 @@ with time quantum set as 100 milliseconds, containers C0, C1, and C2 will run
 for full-time quantum, and container C3 will run for half-time quantum i.e 50
 milliseconds.
 
+The flag `--cpu-quota` is to limit the CPU usage of a container. The default value
+of 0 allows the container to take the full 100000(default cfs_period_us)
+CPU CFS(Complete Fair Scheduler) period or 100% of the cpu resouce(1 CPU). The CFS
+is the default scheduler used by Linux kernel. Set this value to 50000 is
+to limit the container to 50% of the CPU resource(1 CPU). And if there are
+multiple CPUs, the value of `--cpu-quota` could be bigger than 
+1000000(default cfs_period_us)
+
```

Comment:
Mentioning the multiple CPUs is an improvement.  

Some of your sentences are still awkward. In English "a CPU" = "1 CPU" --- include the parenthetical once just for extra precision.  

Mentioning the `cfs_period_us` setting without any explanation of that keyword is confusing.  Leave that reference out. Instead, give the reader a link to the kernel docs.  


Try this instead:

---------

The `--cpu-quota` flag limits the container's CPU usage. The default `0` value allows the container to take 100% of a CPU resource (1 CPU).  The Completely Fair Schedulef (CFS) handles resource allocation for executing  processes and is the default Linux scheduler used by the kernel.  Set this value to `50000` to limit the container to 50% of a CPU resource. For multiple CPUs, adjust the `--cpu-quota` as necessary.  For more information, see the [CFS documentation on bandwidth limiting](https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt).